.github/workflows/ci.yml

```diff
@@ -3,6 +3,8 @@ name: Docker Build
 on:
   push:
     branches: master
+    tags:
+      - v*
```

Comment:
is this for `vx.y.z`? If so it needs a comment, it looks like a typo.

Reply:
Yep, I'll change that.